assets/components/odf-lvm/topolvm_default-storage-class.yaml

```diff
@@ -1,9 +1,13 @@
 apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
-  name: kubevirt-hostpath-provisioner
   annotations:
     storageclass.kubernetes.io/is-default-class: "true"
-provisioner: kubevirt.io/hostpath-provisioner
+  labels:
+  name: topolvm-provisioner
```

Comment:
Is this name significant to the code? Should it be a variation of ODF-LVM to be consistent with branding elsewhere?

Reply:
No, the storageClass name can be arbitrary, as can the deployment and daemonset, and other included APIs.  

WRT branding, on OCP the ODF LVM Operator deploys all the operand APIs (pods, configMaps, secrets, role, roleBindings, etc) with a `topolvm-` prefix.  Functionally, this is a trivial string replacement in rebase automation.

I'm a little concerned about renaming a OCP component for the sake of branding, which could create the appearance of different LVM provisioning solutions between OCP and MicroShift.  I'll defer to your take on that though.

Reply:
If the LVMO uses the `topolvm-` prefix elsewhere, I agree we should stay consistent with that. Thanks for explaining the reasoning here. We should try to mention this decision in the enhancement as well.

Reply:
Maintaining consistency makes sense to me too